src/components/pico-controls.js

```diff
@@ -26,12 +26,12 @@ var PICO_MODEL_GLB_BASE_URL = AFRAME_CDN_ROOT + 'controllers/pico/pico4/';
  */
 var INPUT_MAPPING_WEBXR = {
   left: {
-    axes: {touchpad: [2, 3]},
-    buttons: ['trigger', 'squeeze', 'none', 'thumbstick', 'xbutton', 'ybutton']
+    axes: {thumbstick: [2, 3]},
```

Comment:
Do these apply to all the pico controllers?

Reply:
Yes, in my tests one would not be able to e.g. turn using the blink-controls component and perform gestures with hand-controls without these changes.

Reply:
I mean using wolvic, which advertises the right controller type.

Reply:
What about other browsers? The built-in one?

Reply:
The Pico browser uses the oculus-controls component and behaves as an oculus controller. It was working properly before these changes and keeps doing so afterwards.